### DISCUSSION TITLE
Select between months and days

### DISCUSSION THREAD

**Discussion by mithatercan** [prisma/prisma#14680] [Q&A] [answered]
Hello!
Is it possible to search by month and by day in prisma? I understand how to search by whole date, but I want to pick fields between months and dates.

```javascript

return await prisma.users.findMany({
    where: {
      birthday: {
        // like this, between Jan 05 - Jan 23
        lte: "01-23",
        gte: "01-05",
      },
    },
  });

```

**Comment by austincrim** [comment, marked as answer]
Hey @mithatercan 👋,

It is not possible to query this way with Prisma `DateTime` fields.

You could potentially store month and day separately and then filter on both using an `AND` condition.

**Reply by mithatercan** [reply]
Hello, @austincrim.

Thank you for responding. Working with dates in prisma is a little tricky. I hope it gets better.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
